chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -240,6 +240,7 @@
             variables(
                       :admin_node_ip => admin_ip,
                       :web_port => web_port,
+                      :packages => node[:provisioner][:suse][:autoyast][:packages]["suse-" + target_platform_version.sub(".", "_")],
```

Comment:
Maybe use `os..sub(".", "_")`? (Still unsure we want this underscore there, but let's discuss in the other thread)